package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "calc",
```

This project name does not fit the project at all, it should be something like github-profile-preview or something like that

```suggestion
  "name": "github-profile-preview",
```